easybuild/easyconfigs/b/BinSanity/BinSanity-0.3.3-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,51 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'BinSanity'
+version = '0.3.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/edgraham/BinSanity/wiki'
+description = """BinSanity contains a suite a scripts designed to cluster contigs generated from
+metagenomic assembly into putative genomes."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/edgraham/BinSanity/archive/']
+sources = ['0d529465b5b1fa004e8fb465511041ca5098a764.tar.gz']
+checksums = ['cb72551ff309201b8806b01a50fd1869cc8b9511d889e3e2c290b62f52799be1']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('subread', '1.6.5'),
+    ('scikit-learn', '0.21.3', versionsuffix),
+    ('CheckM', '1.1.2', versionsuffix),
+    ('HMMER', '3.2.1'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+options = {'modulename': False}
```

Comment:
Maybe add a comment to clarify this?

The `site-packages` subdir only contains `Binsanity-0.3.3.dist-info`, so there's nothing to `import`, indeed...